.github/workflows/holy-grail.yml

```diff
@@ -75,14 +84,19 @@ jobs:
 
       - name: Create release branch
         run: |
-          git checkout -b release/tegel@${{ steps.version.outputs.PACKAGE_VERSION }}
+          BRANCH_NAME=release/tegel@${{ steps.version.outputs.PACKAGE_VERSION }}
+          if [[ "${{ github.event.inputs.dryVersion }}" == "true" || "${{ github.event_name }}" == "schedule" ]]; then
+            BRANCH_NAME="$BRANCH_NAME-dry-run"
```

Comment:
Suggestions from call:

1. See if timestamp can be added in name of the branch so script can be run multiple times a day and week without creating naming issues for Git. It should contain year, month,day, hour and minute
2. Consider if in this script, or new one there should be step to delete all branches created by dry run in last 2 days.